src/Lib/Items/CanvasImage.vala

```diff
@@ -103,6 +105,30 @@ public class Akira.Lib.Items.CanvasImage : Goo.CanvasImage, Akira.Lib.Items.Canv
         });
     }
 
+    private bool is_width_small (int width, int height) {
```

Comment:
The `size` component is available anywhere in this file, no need to pass any attribute for these methods.

Reply:
the attributes is for the future because in my next PR I will add this method when the Image size is changed
since the user can resize the Image width to 2 and break the app again

Reply:
I did not add it in this PR for some issues it had but I will add it in my next PR

Reply:
In that case you should rework this and do these calculations in the `Size` component, check if the item is an image and run these checks.